beacon_node/beacon_chain/tests/rewards.rs

```diff
@@ -119,3 +126,187 @@ async fn test_sync_committee_rewards() {
         mismatches.join(",")
     );
 }
+
+#[tokio::test]
+async fn test_verify_attestation_rewards_base() {
+    let harness = get_harness(E::default_spec());
+
+    // epoch 0 (N), only two thirds of validators vote.
+    let two_thirds = (VALIDATOR_COUNT / 3) * 2;
+    let two_thirds_validators: Vec<usize> = (0..two_thirds).collect();
+    harness
+        .extend_chain(
+            E::slots_per_epoch() as usize,
+            BlockStrategy::OnCanonicalHead,
+            AttestationStrategy::SomeValidators(two_thirds_validators),
+        )
+        .await;
+
+    let initial_balances: Vec<u64> = harness.get_current_state().balances().clone().into();
+
+    // extend slots to beginning of epoch N + 2
+    harness.extend_slots(E::slots_per_epoch() as usize).await;
+
+    // compute reward deltas for all validators in epoch N
+    let all_validators: Vec<ValidatorId> = (0..VALIDATOR_COUNT)
+        .map(|idx| ValidatorId::Index(idx as u64))
+        .collect();
+
+    let StandardAttestationRewards {
+        ideal_rewards,
+        total_rewards,
+    } = harness
+        .chain
+        .compute_attestation_rewards(Epoch::new(0), all_validators, null_logger().unwrap())
```

Comment:
Lets clean this up in the func definition by having `compute_attestation_rewards` use the `self.log`. I'm not sure why it wasn't doing that already

```suggestion
        .compute_attestation_rewards(Epoch::new(0), all_validators)
```

Reply:
Good catch!! oh yeah we have the `beacon_chain`! 😅 
Fixed in https://github.com/sigp/lighthouse/pull/4474/commits/629e05049b3463f85cdb265223d119a97a5cf9f4.